var/spack/repos/spack_repo/builtin/packages/quandary/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cached_cmake import CachedCMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
+
+from spack.package import *
+
+
+class Quandary(CachedCMakePackage, CudaPackage, ROCmPackage):
+    """Optimal control for open quantum systems"""
+
+    homepage = "https://github.com/LLNL/quandary"
+    git = "https://github.com/LLNL/quandary.git"
+
+    maintainers("steffi7574", "tdrwenski")
+
+    license("MIT", checked_by="tdrwenski")
+
+    version("main", branch="main", preferred=True)
+    version("learning", branch="learning")
+
+    variant("slepc", default=False, description="Build with Slepc library")
+    variant("debug", default=False, description="Debug mode")
+    variant("test", default=False, description="Add dependencies needed for testing")
+
+    depends_on("cxx", type="build")
```

Comment:
Project CMakeLists.txt has `LANGUAGES CXX C`. I can't tell whether this should translate into a dependency on `"c"` as well. Maybe worth checking.

Reply:
yes, good catch. Quandary is a C++ project but depends on C libraries like PETSc so I think it is correct to say it depends on "C". Fixed in b92e75303c85328a675c34b2d970a43fa84c67f0